dynatrace_topology/README.md

```diff
@@ -0,0 +1,43 @@
+# Agent Check: Dynatrace Topology
+
+## Overview
+
+This check synchronizes [Dynatrace][1] smartscape topology through the StackState Agent. Currently supported Component Types are : 
+* _Hosts_
+* _Applications_
+* _Processes_
+* _Process-Groups_
+* _Services_
+
+## Setup
+
+### Prerequisite
+
+* An API Token from Dynatrace must have access to read the smartscape topology.
```

Comment:
That token must have at least `DataExport`  token permission, I guess?
https://www.dynatrace.com/support/help/dynatrace-api/basics/dynatrace-api-authentication/